rskj-core/src/main/java/org/ethereum/datasource/DataSourceKeyIterator.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2017 RSK Labs Ltd.
+ * (derived from ethereumJ library, Copyright (c) 2016 <ether.camp>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.ethereum.datasource;
+
+import org.ethereum.db.ByteArrayWrapper;
+
+import java.util.*;
+
+public interface DataSourceKeyIterator extends Iterator<ByteArrayWrapper>, AutoCloseable {
+    void seekToFirst();
```

Comment:
is this method only needed to initialise the RockDbIterator or is it used for other reasons?

Reply:
LevelDB and RocksDB iterators

Reply:
Addressed